syne_tune/blackbox_repository/conversion_scripts/scripts/yahpo_import.py

```diff
@@ -184,6 +194,8 @@ def _objective_function(
                 ), f"fidelity = {fidelity_value} not contained in {self.fidelity_values}"
                 fidelity = {k: fidelity_value / 20}
             configuration.update(fidelity)
+            active_hps = self.active_hyperparameters(configuration)
```

Comment:
As far as I know, active hyperparameters are used for conditional search spaces, rbv2_xgboost does indeed have inactive HPs.

Reply:
OK, so that could really explain these weird results I got. Cool.

Let us bring this in, and then I can rerun the experiments I ran and see whether the issues go away.